bodhi/models/models.py

```diff
@@ -714,6 +723,12 @@ def new(cls, request, data):
                 'description': 'Auto-karma stable requests disabled.',
             })
 
+        elif data['autokarma']:
+        # We have to set autokarma column to True here.
```

Comment:
The web interface has "Auto-request stable?" enabled by default, so I think we can have `autokarma` default to `True`.